e2e_testing/xfail_sets.py

```diff
@@ -97,6 +93,13 @@
     # ERROR: RuntimeError: Found a custom (non-ATen) operator that either mutates or its inputs: prims::squeeze.. Getting these operators to work with functionalization requires some extra work. For mutable ops you need to register a corresponding out-of-place variant of the op, and you also need to register a Functionalization kernel that performs some boilerplate, telling functionalization to map from the mutable op to the out-of-place op. See a more complete example of how to do this at https://gist.github.com/bdhirsh/7dadbf6296f8f7d1abcf4c482f438aaa.
     "PrimsSqueezeModule_basic",
     "PrimsSqueezeEmptyDimensionsModule_basic",
+
+    # https://github.com/pytorch/pytorch/blob/master/torch/_dynamo/symbolic_convert.py#L1972
```

Comment:
can you file an upstream bug for this?

Also this looks like a dynamo issue -- I don't understand how this worked with the previous code but fails with the new code.

Reply:
These aren't upstream bugs, these are false negative tests in torch-mlir; see [false_negatives.py](https://github.com/llvm/torch-mlir/pull/1996/files#diff-4f5914874fa7aa4c05ae28dbbf8b6468f612a1885ce226cd3fab03cdd3b84ffa) where you have two examples (`ViewCollapseDynamicWithAtenSizeIntModule` and `NeFloatIntModule`) of tests that should fail because torch-mlir returns the wrong thing but nonetheless pass because there's a graph break and dynamo punts to runtime eval:

```
m = ViewCollapseDynamicWithAtenSizeIntModule()
o = run(m, (torch.rand(2, 3, 5, 4, 12, 32), torch.tensor(3), torch.tensor(5)))
assert isinstance(o, torch.Tensor)
assert o.shape == (2, 3, 5, 4, 384)
```

but `print(type(r))` inside of `_refbackend_torchdynamo_backend` prints `<class 'tuple'>` corresponding to an fx graph like this

```
graph():
    %arg0_1 : [#users=1] = placeholder[target=arg0_1]
    %arg1_1 : [#users=1] = placeholder[target=arg1_1]
    %arg2_1 : [#users=1] = placeholder[target=arg2_1]
    %clone : [#users=1] = call_function[target=torch.ops.aten.clone](args = (%arg0_1,), kwargs = {})
    %clone_1 : [#users=1] = call_function[target=torch.ops.aten.clone](args = (%arg1_1,), kwargs = {})
    %clone_2 : [#users=1] = call_function[target=torch.ops.aten.clone](args = (%arg2_1,), kwargs = {})
    return (clone, clone_1, clone_2)
```

Dynamo break the graph for an unrelated reason:

```
[2023-04-14 17:23:35,811] torch._dynamo.symbolic_convert: [WARNING] Graph break: call_function BuiltinVariable(int) [TensorVariable()] {} from user code at   File "false_negative.py", line 93, in <lambda>
    f = lambda method, *inputs: method(*inputs)
  File "false_negative.py", line 90, in item_symbol_that_clones_inputs
    result = model.forward(*cloned_inputs)
  File "false_negative.py", line 104, in forward
    return a.view(a.size(0), int(b), int(c), a.size(3), 384)
```

cc @powderluv

Reply:
putting 

```python
                if True:
                    raise Exception("raising break")
```

[here](https://github.com/pytorch/pytorch/blob/e2923b521bd7fc67fa1e8f404397b20d1ec81668/torch/_dynamo/symbolic_convert.py#L358-L359) in dynamo gives you the real number of fails:

```
Summary:
    Passed: 750
    Failed: 57
    Expectedly Failed: 38
```

Reply:
Yes, this seems like a Dynamo bug -- Dynamo should trace a graph but bails out, so it is missing feature work in Dynamo. I would suggest that we run the tests with nopython=True and file any failures as upstream Dynamo bugs.